buckets.yaml

```diff
@@ -10,9 +10,9 @@
 gs://kubernetes-jenkins/logs/:
   contact: "fejta"
   prefix: ""
-gs://kube_azure_log/:
-  contact: "travisnewhouse"
-  prefix: "azure:"
+#gs://kube_azure_log/:
```

Comment:
maybe just delete these rather than comment out? we can always revert this change later.